cli/format/sort.go

```diff
@@ -34,3 +36,17 @@ func (s ByStartsAt) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 func (s ByStartsAt) Less(i, j int) bool {
 	return time.Time(*s[i].StartsAt).Before(time.Time(*s[j].StartsAt))
 }
+
+type ByAddress []*models.PeerStatus
+
+func (s ByAddress) Len() int      { return len(s) }
+func (s ByAddress) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
+func (s ByAddress) Less(i, j int) bool {
+	ip1, port1, _ := net.SplitHostPort(*s[i].Address)
+	ip2, port2, _ := net.SplitHostPort(*s[j].Address)
+	if ip1 == ip2 {
+		return port1 < port2
```

Comment:
you need to convert them to integer for comparison
```suggestion
                p1, _ := strconv.Atoi(port1)
                p2, _ := strconv.Atoi(port2)
		return p1 < p2
```

Reply:
sorry about that added.